packages/orbit-components/src/NavigationBar/index.tsx

```diff
@@ -60,17 +66,40 @@ const NavigationBar = ({
     };
   });
 
+  React.useEffect(() => {
+    const currentScrollPosition =
+      window.scrollY ||
+      window.pageYOffset ||
+      (document.documentElement && document.documentElement.scrollTop);
+    if (transparentBgAtTop) {
+      setTransparentBg(currentScrollPosition <= 0);
```

Comment:
shouldn't `== 0` be enough here?

Reply:
I remember checking something like this, yes :)